internal/driver/fetch.go

```diff
@@ -107,6 +137,36 @@ func fetchProfiles(s *source, o *plugin.Options) (*profile.Profile, error) {
 	return p, nil
 }
 
+func grabSourceAndBases(sources, bases []profileSource, fetch plugin.Fetcher, obj plugin.ObjTool, ui plugin.UI) (*profile.Profile, *profile.Profile, plugin.MappingSources, plugin.MappingSources, bool, int, int, error) {
```

Comment:
Should the name of the function be grabSourcesAndBases as input is a list of sources?